README.md

```diff
@@ -37,7 +37,7 @@ conda install -c conda-forge conda-build
 conda build recipe -c conda-forge --output-folder dist/conda --no-anaconda-upload
 
 # Install the built `coiled-runtime` metapackage
-conda install -c conda-forge -c ./dist/conda/ coiled-runtime
+conda install -c ./dist/conda/ coiled-runtime
```

Comment:
I think we want to keep `-c conda-forge` to make sure we can install dependencies like `numpy`, `pandas`, etc. Though you probably need to make `conda-forge` lower priority than `coiled` as `coiled-runtime` now exists on `conda-forge`